docs/get-started/create-a-project.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_label: 'Create a project'
+sidebar_position: 2
+---
+
+# Create a project
+
+First, create a project, move to it and initialize Git.
+```shell
+$ git init
+
+Initialized empty Git repository
+```
+
+After Git we also initialize DVC.
```

Comment:
Is it really relevant to use DVC in this tutorial? Assets are only ~20 MB of size (GH can handle gigabytes!), and while most users will be familiar with `git`, `dvc` will probably put an unnecessary burden on the setting up phase, while mostly only users who already know `dvc` should in fact use it.

Reply:
Removed dvc allusions in https://github.com/polyphene/docs.holium.org/pull/16/commits/dbdbdcd599965c55bab1f49a2c21004793b11591